misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java

```diff
@@ -130,32 +126,55 @@ static void execute(List<String> argList, boolean helpFlag, boolean dryRun, Path
                         }
                     }
 
-                    // Check whether the given directory is not in a ballerina project.
-                    if (!FormatUtil.isBallerinaProject(sourceRootPath)) {
-                        throw LauncherUtils.createLauncherException(Messages.getNotBallerinaProject());
-                    }
-                    BLangPackage bLangPackage = FormatUtil
-                            .compileModule(sourceRootPath, getModuleName(moduleName));
+                    BuildProject project = BuildProject.load(sourceRootPath, constructBuildOptions());
+                    Package currentPackage = project.currentPackage();
+                    Module module = currentPackage.module(ModuleName.from(currentPackage.packageName(),
```

Comment:
We need to iterate the module ids and get the module

Reply:
This is for formatting a single module that is provided as an argument to the formatter command. This approach should be okay right?

Reply:
yes ok, I mistook it